github/github.go

```diff
@@ -160,6 +161,15 @@ type NewIssueOptions struct {
 	Labels    []string // List of labels to apply. They will be created if new
 }
 
+// UpdateReleasePageOptions is a struct of optional fields for creating/updating releases
+type UpdateReleasePageOptions struct {
+	Name       string // name of the release
+	Body       string // content/body of the release
+	Draft      bool   // Mark release as draft if true
+	Prerelease bool   // Mark release as pre-release if true
+	Latest     bool   // Mark release as latest (at the time of update) if true
```

Comment:
Let's please use Godoc comments here (e.g. `Name is ...`).

Reply:
done!